ui-modules/utils/catalog-uploader/catalog-uploader.js

```diff
@@ -120,7 +121,9 @@ export function catalogUploaderDirective($compile, brooklynCatalogUploader) {
         };
 
         scope.getCatalogItemUrl = (item)=> {
-            return item.supertypes.includes('org.apache.brooklyn.api.location.Location')
+            let itemTraits = item.tags? item.tags.find(item => item.hasOwnProperty("traits")) : {"traits":[]};
+            return (item.supertypes ? item.supertypes : itemTraits.traits)
+                .includes('org.apache.brooklyn.api.location.Location')
```

Comment:
Why is this change for? It doesn't look related to the file uploader

Reply:
@tbouron  Strictly not, but it was a minor bug we corrected at the same time. Can make it into a separate PR if required.

Reply:
@frogfather Leaving it there is fine but it would be great if you can explain why this is for

Reply:
@tbouron Added an explanation in the description of the PR. Do you also want comments in the code?